DirectEmployers/MyJobs
# [PD-3463] cleanup table components

## What's this PR do?
PR makes minor cleanups to the table components

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application or view the code

## How should this be manually tested?
Pull down the repository. Go into any analytics application and make sure the table is still functional. You can also view the code changes

## Any background context you want to provide?
